components/tikv_util/src/metrics/allocator_metrics.rs

```diff
@@ -13,6 +13,7 @@ pub fn monitor_allocator_stats<S: Into<String>>(namespace: S) -> Result<()> {
 struct AllocStatsCollector {
     descs: Vec<Desc>,
     memory_stats: IntGaugeVec,
+    thread_stats: IntGaugeVec,
```

Comment:
Should we add a metric about the number of arenas?

Reply:
+1.